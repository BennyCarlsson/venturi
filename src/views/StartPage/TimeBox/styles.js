import styled from 'styled-components';
import Colors from 'tokens/colors';
import { ReactComponent as MapPinLogo } from 'assets/icons/map-pin.svg';

export const TimeBoxWrapper = styled.div`
  padding: 16px 32px;
  margin: 48px 24px;
  background: ${Colors.white};
  border: 1px solid #f3f3f3;
  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.1);
`;

export const TimeLocationWrapper = styled.div`
  display: flex;
  flex-direction: column;
  margin-bottom: 16px;
`;

export const TimeWrapper = styled.div``;

export const LocationWrapper = styled.div`
  display: flex;
  align-items: center;
`;

export const StyledMapPinLogo = styled(MapPinLogo)`
  height: 22px;
  width: 18px;
  padding-right: 16px;
  stroke: ${Colors.vBlue};
`;

export const NumberAndNameWrapper = styled.div`
  display: flex;
`;

export const NumberBox = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  margin-right: 16px;
  border-radius: 4px;
  height: 20px;
  width: 20px;
  background-color: ${Colors.vBlack};
`;
