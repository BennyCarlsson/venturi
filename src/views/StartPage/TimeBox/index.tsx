import Heading from 'components/Heading';
import React from 'react';
import colors from 'tokens/colors';
import { FontWeight } from 'types';
import { TimeBoxWrapper } from './styles';

type TimeBoxProps = {
  title: string;
};

const TimeBox = ({ title, ...props }: TimeBoxProps) => {
  return (
    <TimeBoxWrapper {...props}>
      {
        <Heading
          headingType={'h1'}
          fontWeight={FontWeight.medium}
          fontSize={18}
          color={colors.white}
        >
          Title
        </Heading>
      }
    </TimeBoxWrapper>
  );
};

export default TimeBox;
