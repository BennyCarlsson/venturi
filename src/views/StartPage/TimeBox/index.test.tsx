import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import TimeBox from '.';

describe('<TimeBox />', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<TimeBox title="test" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should renders without crashing', () => {
    render(<TimeBox title="test" />);
    expect(screen).toBeTruthy();
  });
});
