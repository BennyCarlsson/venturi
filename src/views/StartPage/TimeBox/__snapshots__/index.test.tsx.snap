// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TimeBox /> should match snapshot 1`] = `
<div
  className="sc-gKsewC fJFOVJ"
>
  <h1
    className="sc-bdfBwQ UZuHf"
    color="#FFFFFF"
    fontSize={18}
    fontWeight={500}
  >
    Title
  </h1>
</div>
`;
